payments-core/src/main/java/com/stripe/android/model/PaymentMethodCreateParams.kt

```diff
@@ -1098,6 +1098,23 @@ data class PaymentMethodCreateParams internal constructor(
             )
         }
 
+                /**
+         * Helper method to create [PaymentMethodCreateParams] with [PaymentMethod.Type.Sunbit] as the payment
```

Comment:
do you know why this is showing the fully qualified type name here (`PaymentMethod.Type.Sunbit`)? Can it show `Sunbit` instead?

Reply:
At the time of this comment, we didn't have a `Sunbit` method. I added it in here (and piped it through in a few places), but I'm not sure if we get much value out of it.

Here's the [commit](https://github.com/stripe/stripe-android/pull/8595/commits/2455f3602e32c06cac9fd429afa142e5c1b7466d), let me know if you think we should take this approach for the other payment methods.

Reply:
Hm agreed that it doesn't seem like we get much value out of it and it looks like we didn't do this for Multibanco or Amazon Pay, so I don't think we need it here either or for the other payment methods. Sorry for the churn!